Makefile

```diff
@@ -125,7 +139,10 @@ check-certs: # Check cert subject
 	openssl x509 -noout -in /tmp/cert.pem --subject
 
 ##@ Build
-build: generate fmt vet ## Build manager binary.
+build: generate fmt vet gosec imports ## Build manager binary.
```

Comment:
invoking gosec causes a failure in the CI since it specifies `-it` flag in line 87 https://github.com/project-flotta/flotta-operator/pull/178/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R88
```
docker run --rm -it -v /home/runner/work/flotta-operator/flotta-operator:/opt/data/:z docker.io/securego/gosec -exclude-generated /opt/data/...
the input device is not a TTY
```
 pls remove that flag